.github/mongo_setup.sh

```diff
@@ -0,0 +1,15 @@
+docker run -d -p 27001:27001 -p 27002:27002 -p 27003:27003 --name mongo -e "REPLICA_SET_NAME=rs0" zxilly/mongodb-rs
+
+echo "Waiting for MongoDB to accept connections"
+TIMER=0
+until docker exec --tty mongo mongo --port 27001 --eval "db.serverStatus()"
```

Comment:
Does this also ensure that the replicaset is up and running?
This is something that [this](https://github.com/node-casbin/mongoose-adapter/pull/28/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L53) tried to achieve.

Reply:
I make another solution, plz review.

Reply:
@Zxilly

Reply:
@Sefriol